src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -133,6 +133,16 @@ export class SqlSessionManager implements nb.SessionManager {
 		}
 		return Promise.resolve();
 	}
+
+	shutdownAll(): Thenable<void> {
+		return Promise.all(SqlSessionManager._sessions.map(session => {
+			return this.shutdown(session.id);
+		})).then();
+	}
+
+	dispose(): void {
```

Comment:
Do we still need to include dispose() in the extension APIs if it's a no-op?

Reply:
Dispose() is needed for the jupyterSessionManager but no-op for sqlSessionManager.